From: Bruce Rogers <brogers@suse.com>
Date: Mon, 14 Sep 2020 14:15:00 -0600
Subject: Revert "roms/efirom, tests/uefi-test-tools: update edk2's own
 submodules first"

This reverts commit ec87b5daca761039bbcf781eedbe4987f790836f.

No need. In our build system submodules are checked out.

Signed-off-by: Bruce Rogers <brogers@suse.com>
[DF: Rebased on top of 6.2.0]
---
 roms/Makefile                  | 2 --
 tests/uefi-test-tools/Makefile | 1 -
 2 files changed, 3 deletions(-)

diff --git a/roms/Makefile b/roms/Makefile
index 3d53bd528666b594ab3d6b631450..c7c6e973db59572e8b8bee74e969 100644
--- a/roms/Makefile
+++ b/roms/Makefile
@@ -151,8 +151,6 @@ build-efi-roms: build-pxe-roms
 #    efirom
 #
 edk2-basetools:
-	cd edk2/BaseTools && git submodule update --init --force \
-		Source/C/BrotliCompress/brotli
 	$(MAKE) -C edk2/BaseTools \
 		PYTHON_COMMAND=$${EDK2_PYTHON_COMMAND:-python3} \
 		EXTRA_OPTFLAGS='$(EDK2_BASETOOLS_OPTFLAGS)' \
diff --git a/tests/uefi-test-tools/Makefile b/tests/uefi-test-tools/Makefile
index 471f0de981090b3d419b8bc7890c..1dcddcdbbabf1c7dbea9c233d0ef 100644
--- a/tests/uefi-test-tools/Makefile
+++ b/tests/uefi-test-tools/Makefile
@@ -99,7 +99,6 @@ Build/bios-tables-test.%.efi: build-edk2-tools
 	+./build.sh $(edk2_dir) BiosTablesTest $* $@
 
 build-edk2-tools:
-	cd $(edk2_dir)/BaseTools && git submodule update --init --force
 	$(MAKE) -C $(edk2_dir)/BaseTools \
 		PYTHON_COMMAND=$${EDK2_PYTHON_COMMAND:-python3} \
 		EXTRA_OPTFLAGS='$(EDK2_BASETOOLS_OPTFLAGS)' \
